audit-service/src/component-tests/java/uk/gov/gchq/palisade/component/audit/request/AuditSuccessMessageTest.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package uk.gov.gchq.palisade.service.audit.request;
```

Comment:
This is a unit test instead of a component test.  Think it should stay in the unit-test package.

Reply:
Decided to be component, due to how the spring context handles json (de)serializers